components/campaign/actions.ts

```diff
@@ -55,6 +55,7 @@ export function newCampaign(
   difficulty: CampaignDifficulty,
   deckIds: number[],
   chaosBag: ChaosBag,
+  chaosBagResults: ChaosBagResults,
```

Comment:
Was the places where `newCampaign` is called updated as well.

My guess is any 'new' campaign is going to always have an empty set of 'drawn' chaosBag tokens, so maybe we just leave this out and put a default down below in what gets dispatched?

Reply:
Yeah, I see the default is inserted below, maybe this `newCampaign` edit isn't needed anymore?